docs/README.md

```diff
@@ -2,6 +2,11 @@
 
 ## Overview
 
-This folder contains documents that relate to the project.
+The docs folder contains two subfolders - `user` and `contributor`. 
+The `user` subfolder contains the end-user documentation. Depending on your module needs, the documentation must include overview, usage, or technical reference documents. The content of this folder is displayed on the [Kyma website](https://kyma-project.io/#/). To display the content on the website properly, you must add your documentation to the `_sidebar.md` file that must be created in the `user` folder. For more information on how to use `_sidebar.md`, follow [this guide](https://github.com/kyma-project/community/blob/main/docs/guidelines/content-guidelines/01-user-docs.md#publish-a-document-from-a-new-module-repostory).
 
-> In this document, provide the list of all documents that this folder contains, together with links to those documents and short information on what they describe.
+The `contributor` subfolder includes any developer-related documentation to help them manually install, develop, and operate a module.
+
+All documents must be properly numbered according to the structure proposed in [this decision record](https://github.tools.sap/kyma/community/issues/180) to have a common structure across all modules.
+
+You can divide your documentation into relevant folders to avoid many documents in one `docs/user` or `docs/contributor` folder. For example, if you have many technical reference documents, you can create a `technical reference` folder in `docs/user` and keep your documentation there. Each folder must have its own `_sidebar.md` file with the links to the main module page and the docs in the given folder.
```

Comment:
```suggestion
You can divide your documentation into more subfolders to avoid having too many documents in one `docs/user` or `docs/contributor` folder. For example, if you have many technical reference documents, you can create a `technical reference` subfolder in `docs/user` and keep relevant documentation there. Each subfolder in the `user` folder must have its own `_sidebar.md` file with the links to the main module page and the list of docs it contains.
```

Reply:
or maybe the previous sentence only refers to the numbering system?